librz/core/cfile.c

```diff
@@ -43,7 +43,7 @@ static void loadGP(RzCore *core) {
 		ut64 gp = rz_num_math(core->num, "loc._gp");
 		if (!gp || gp == UT64_MAX) {
 			rz_config_set(core->config, "analysis.roregs", "zero");
-			rz_core_cmd0(core, "10aes@entry0");
+			rz_core_cmd0(core, "s entry ; 10 aes");
```

Comment:
Could you please use the API? Both have one.

Reply:
See `rz_core_seek_opt(core, n, true, !silent);` and `RZ_API bool rz_core_debug_step_one(RzCore *core, int times)`

Reply:
Or just `rz_core_seek(core, addr, false);`